### DISCUSSION TITLE
Realtime chat

### DISCUSSION THREAD

**Discussion by ciaranmcnulty** [Behat/Behat#1493] [General]
It would be good to get something like a discord going, I'd love to talk more 'in person' with the new maintainers

There is a Cucumber community discord, we can almost certainly have one or more channels in there and it'd be good to hook up with the 'other cucumber' maintainers too in the general channels

Alternative suggestions welcomed!

**Comment by ciaranmcnulty** [comment]
See https://github.com/Behat/Behat/pull/1492

**Comment by carlos-granados** [comment]
@ciaranmcnulty Never really used Discord before but ready to give it a go. We also mentioned the possibility of using Gitter as Behat already has a Gitter channel (or however they are called there) https://gitter.im/Behat/Behat.

**Comment by acoulton** [comment]
Sounds like a good idea. I've never used Discord, my kids would definitely find that amusing :rofl: - happy with whatever platform works for everyone!

**Comment by carlos-granados** [comment]
@ciaranmcnulty do you think you might set up the Discord channel and we'll give it a go?

**Comment by ttomdewit** [comment]
I’m game with Discord as long as we share most important stuff out in the open. Don’t want it to become a support/FAQ area as we do not know when they might close it, start charging etc.
